docs/config.json.md

```diff
@@ -43,6 +43,7 @@ This README provides an explanation for each field in the JSON configuration fil
 
 ### MillisecondsPerBlock
 - **MillisecondsPerBlock**: Time interval between blocks in milliseconds. Default is `15000` (15 seconds).
```

Comment:
Still this will be the easiest way for private nets otherwise we will need a round of multi sig decision.

So, better to explain that this is still configurable